markets/perps-market/contracts/modules/MarketConfigurationModule.sol

```diff
@@ -24,6 +24,21 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
         config.settlementStrategies.push(strategy);
     }
 
+    function setSettlementStrategyEnabled(
+        uint128 marketId,
+        SettlementStrategy.Type strategyType,
+        bool isEnabled
+    ) external override {
+        PerpsMarket.load(marketId).onlyMarketOwner();
+        PerpsMarketConfiguration.Data storage config = PerpsMarketConfiguration.load(marketId);
+
+        for (uint256 index = 0; index < config.settlementStrategies.length; index++) {
+            if (config.settlementStrategies[index].strategyType == strategyType) {
+                config.settlementStrategies[index].disabled = isEnabled;
+            }
+        }
+    }
```

Comment:
use the logic from here: https://github.com/Synthetixio/synthetix-v3/blob/e841287cf55ca5b687ce6e8660d31da81b4253d4/markets/spot-market/contracts/modules/AsyncOrderConfigurationModule.sol

you can directly access the settlement straetgy via the index